PORT-2154 Some fixes to platform overview (#115)
port-labs/port-docs: docs/platform-overview.md

Visit our [Swagger.](https://api.getport.io/static/index.html#/)
:::

> We will now list the core components that make up Port with short explanations and links to the comprehensive documentation of each component:

Reviewer: ```suggestion
We will now list the core components that construct Port, with short explanations and links to the comprehensive documentation of each component:
```

## Software Catalog

> - [**Blueprints**](./software-catalog/blueprint/) are our basic building blocks in Port. They represent assets that can be managed in Port, such as `Microservice`, `Environments`, `Packages`, `Clusters`, `Databases` and many more;
> - [**Relations**](./software-catalog/relation/) enable us to make connections between Blueprints, doing so provides logical context to the Software Catalog;
> - [**Entities**](./software-catalog/entity/) are objects that match the types defined by Blueprints;
> - [**Widgets**](./software-catalog/widgets/) and [**Dashboards**](./software-catalog/dashboards/) allow you to extend your Software Catalog with convenient and intuitive visual representations of data;

Reviewer: ```suggestion
- [**Blueprints**](./software-catalog/blueprint/) our basic building blocks in Port. They represent assets that can be managed in Port, such as `Microservice`, `Environments`, `Packages`, `Clusters`, `Databases`, and many more;
- [**Relations**](./software-catalog/relation/) enable us to create connections and link between Blueprints, doing so provides logical context to the Software Catalog;
- [**Entities**](./software-catalog/entity/) are objects that match the types defined by Blueprints;
- [**Widgets**](./software-catalog/widgets/) and [**Dashboards**](./software-catalog/dashboards/) allow you to extend your Software Catalog with convenient and intuitive visual representations of data.
```

:::tip
To learn more about the components listed above and more, refer to the [Software Catalog](./software-catalog/) page.
:::

> ## Self-Service Actions

Reviewer: ```suggestion
## Self-service actions
```

> In Port, you can make your Software Catalog active by defining Self-Service Actions for your developers to perform, without assistance or dependency on DevOps teams.

Reviewer: ```suggestion
In Port, you can activate Self-Service Actions for your developers to perform from your software catalog, without assistance or dependency on DevOps teams.
```

  You can listen to a Kafka Topic with any code platform you wish to use, and also use it as a trigger for a serverless function. For example, AWS Lambda.

:::tip
To learn more about the Self-Service Actions and more, refer to the [Self-Service Actions](./self-service-actions/) page and the [Self-Service Actions Deep Dive](./self-service-actions/self-service-actions-deep-dive.md).
:::

## Integrations

In addition to Port's core components, we provide you with multiple integrations, exporters and data ingestion methods:

- Via the **UI**;
- Using our **API**;
- Using 3rd party app integrations, for example [GitHub App](./api-providers/gitops/), [GitHub Action](./api-providers/github-action.md), [K8s Exporter](./exporters/k8s-exporter/) and [Terraform Provider](./api-providers/terraform.md).

:::tip
Refer to Port's API [providers](./api-providers/) and [exporters](./exporters/) pages to learn more.
:::

## Management

Port provides enterprise-grade management tools and integrations to keep track of your Software Catalog and provide access to large R&D teams

> ### The Audit Log

Reviewer: ```suggestion
### The audit log
```

In Port, we store every event in a visual log that shows which events took place on different Blueprints and Entities. For example: which Entity was modified, who invoked a Self-Service Action, what is an event’s status and more.

> ### The Users page and the Teams page

Reviewer: ```suggestion
### The user and team profile page
```

Port has pages dedicated for managing all users and teams in the system:

> - The Users page shows a list of all users, including their roles and group membership;
> - The Teams page shows a list of all teams, including a list of team members for each team.

Reviewer: ```suggestion
- User profile page: shows a list of all users, including their roles and group membership;
- Team profile page: shows a list of all teams, including a list of team members for each team.
```

> #### Managing Users and Teams

Reviewer: ```suggestion
#### Managing users and teams
```
